src/components/check/checkSubscribe.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import { Checkbox } from 'react-native-paper'
+import styles from '../cajaTextoVisible/styles'
+
+export const CheckSubscribe = ({checkout}) => {
+  // const [checked, setChecked] = React.useState(false)
+  // console.log(user)
```

Comment:
Do not leave code comments on GitHub.

Reply:
We have fixed the observations you indicated, we can approve the PR.